### PR TITLE
Add ppo network epoch parameters

### PR DESCRIPTION
**Pull request by jamescasbon**
Allows configuration of training epochs for actor/critic networks in train loop of PPO agent

### CODE REVIEW

**Comment by jamescasbon** [comment]
I merged the head of master and tried following the documentation, but it generates a huge patch by moving the js includes around, minor formatting changes.

Any suggestions?

```
diff --git a/docs/features/benchmarks.html b/docs/features/benchmarks.html
index 1bbd896..569df03 100644
--- a/docs/features/benchmarks.html
+++ b/docs/features/benchmarks.html
@@ -17,13 +17,21 @@
   
 
   
-
-  
+  <script type="text/javascript" src="../_static/js/modernizr.min.js"></script>
   
     
+      <script type="text/javascript" id="documentation_options" data-url_root="../" src="../_static/documentation_options.js"></script>
+        <script type="text/javascript" src="../_static/jquery.js"></script>
+        <script type="text/javascript" src="../_static/underscore.js"></script>
+        <script type="text/javascript" src="../_static/doctools.js"></script>
+        <script type="text/javascript" src="../_static/language_data.js"></script>
+        <script async="async" type="text/javascript" src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.5/latest.js?config=TeX-AMS-MML_HTMLorMML"></script>
+    
+    <script type="text/javascript" src="../_static/js/theme.js"></script>
 
-  
+    
 
+  
   <link rel="stylesheet" href="../_static/css/theme.css" type="text/css" />
   <link rel="stylesheet" href="../_static/pygments.css" type="text/css" />
   <link rel="stylesheet" href="../_static/css/custom.css" type="text/css" />
@@ -33,21 +41,16 @@
     <link rel="prev" title="Environments" href="environments.html" />
     <link href="../_static/css/custom.css" rel="stylesheet" type="text/css">
 
-
-  
-  <script src="../_static/js/modernizr.min.js"></script>
-
 </head>
 
 <body class="wy-body-for-nav">
 
    
   <div class="wy-grid-for-nav">
-
     
     <nav data-toggle="wy-nav-shift" class="wy-nav-side">
       <div class="wy-side-scroll">
-        <div class="wy-side-nav-search">
+        <div class="wy-side-nav-search" >
           
 
           
@@ -193,11 +196,11 @@
 <p>Reinforcement learning is a developing field, and so far it has been particularly difficult to reproduce some of the
 results published in the original papers. Some reasons for this are:</p>
 <ul class="simple">
-<li>Reinforcement learning algorithms are notoriously known as having an unstable learning process.
-The data the neural networks trains on is dynamic, and depends on the random seed defined for the environment.</li>
-<li>Reinforcement learning algorithms have many moving parts. For some environments and agents, there are many
+<li><p>Reinforcement learning algorithms are notoriously known as having an unstable learning process.
+The data the neural networks trains on is dynamic, and depends on the random seed defined for the environment.</p></li>
+<li><p>Reinforcement learning algorithms have many moving parts. For some environments and agents, there are many
 “tricks” which are needed to get the exact behavior the paper authors had seen. Also, there are <strong>a lot</strong> of
-hyper-parameters to set.</li>
+hyper-parameters to set.</p></li>
 </ul>
 <p>In order for a reinforcement learning implementation to be useful for research or for data science, it must be
 shown that it achieves the expected behavior. For this reason, we collected a set of benchmark results from most
@@ -220,7 +223,7 @@ benchmarks stay intact as Coach continues to develop.</p>
         <a href="../selecting_an_algorithm.html" class="btn btn-neutral float-right" title="Selecting an Algorithm" accesskey="n" rel="next">Next <span class="fa fa-arrow-circle-right"></span></a>
       
       
-        <a href="environments.html" class="btn btn-neutral" title="Environments" accesskey="p" rel="prev"><span class="fa fa-arrow-circle-left"></span> Previous</a>
+        <a href="environments.html" class="btn btn-neutral float-left" title="Environments" accesskey="p" rel="prev"><span class="fa fa-arrow-circle-left"></span> Previous</a>
       
     </div>
   
@@ -246,27 +249,16 @@ benchmarks stay intact as Coach continues to develop.</p>
   
 
 
-  
-
-    
-    
-      <script type="text/javascript" id="documentation_options" data-url_root="../" src="../_static/documentation_options.js"></script>
-        <script type="text/javascript" src="../_static/jquery.js"></script>
-        <script type="text/javascript" src="../_static/underscore.js"></script>
-        <script type="text/javascript" src="../_static/doctools.js"></script>
-        <script type="text/javascript" src="../_static/language_data.js"></script>
-        <script async="async" type="text/javascript" src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.5/latest.js?config=TeX-AMS-MML_HTMLorMML"></script>
-    
-
-  
-
-  <script type="text/javascript" src="../_static/js/theme.js"></script>
-
   <script type="text/javascript">
       jQuery(function () {
           SphinxRtdTheme.Navigation.enable(true);
       });
-  </script> 
+  </script>
+
+  
+  
+    
+   
 
 </body>
 </html>

**Comment by galnov** [comment]
This might be due to a new version of one of the sphinx packages. If the html files look ok (you can take a look at it locally), feel free to commit the changes.

**Comment by galnov** [comment]
Closing due to no response.
Please reopen and update the documentation if this change is still needed.
